### ISSUE TITLE
OnNavigatingTo fired twice in XF Prism 7 

### ISSUE THREAD

**Issue by fabcall** [PrismLibrary/Prism#1733]
### Description
Hi, I've been working on an application and noticed that when working with a tabbed page with a single ViewModel (no VM on its children), the OnNavigatingTo method is invoked N + 1 times (one for each child, plus one for parent), where N is the number of child pages. A similar issue can be found here (https://github.com/PrismLibrary/Prism/issues/1302), but for the OnNavigatedTo method.

### Steps to Reproduce

1. Create a TabbedPage and associate a ViewModel to it with some OnNavigatingTo logic.
2. Add pages to TabbedPage Children content without any associated ViewModel.
3. Navigate to TabbedPage and observe the OnNavigatingTo logic being called many times.

### Expected Behavior
The method OnNavigatingTo should be called only once, for the parent TabbedPage.

### Actual Behavior
The method OnNavigatingTo is called as many times as its number of child pages.

### Basic Information

- Version with issue: 7.1.0.431
- Xamarin.Forms version: 3.6.0.220655
- IDE: Visual Studio Community 2017 15.9.7

### Reproduction Link
[BlankApp1-1.zip](https://github.com/PrismLibrary/Prism/files/2952971/BlankApp1-1.zip)

**Comment by dansiegel** [comment]
closing as it is a duplicate of and will be resolved by #1724
